maestro/demos/workflows/ibm-summary.ai/agents.yaml

```diff
@@ -0,0 +1,174 @@
+apiVersion: maestro/v1alpha1
+kind: Agent
+metadata:
+  name: slack
+  labels:
+    app: slack-example
+    custom_agent: slack_agent
+spec:
+  model: dummy
+  framework: custom
+  mode: remote
+  description: slack agent
+  instructions: post a message to slack
+
+---
+apiVersion: maestro/v1alpha1
+kind: Agent
+metadata:
+  name: Paper Finder
+  labels:
+    app: mas-example 
+spec:
+  model: llama3.1
+  framework: beeai
+  mode: remote
+  description: "Retrieve IBM related papers from a certain topic in a certain time frame"
+  instructions: |
+    Input:
+          • category (string) # e.g. "quantum-ph"
+          • since_days (int)  # e.g. "1" for the last 1 days since today's date.
+
+        Task:
+          1. Call the ibm-arXiv tool to search for all papers on `category` published in the last `since_days`. All the functionality is already implemented in the funciton, simply execute the function with the given parameters as the input, and then return the output (which should be titles of research papers) into a list.
+          Directly print out the list in the output.
+  code: |
```

Comment:
One of the things we need to work on is simplifying how these custom tools are added ...